server/server.go

```diff
@@ -81,9 +83,18 @@ func Start(options options.Options) (stop func(), err error) {
 		return nil, err
 	}
 	recorder := broadcaster.NewRecorder(scheme, corev1.EventSource{Component: "metacontroller"})
+
+	runtimeContext := common.RuntimeContext{
```

Comment:
Do you think it makes sense to have a `NewRuntimeContext` function in the common package which will contain the initialization logic from [these](https://github.com/metacontroller/metacontroller/blob/master/server/server.go#L57-L83) lines and return an initialized `RuntimeContext`?